packages/flutter_tools/templates/plugin/.idea/libraries/Dart_SDK.xml.tmpl

```diff
@@ -0,0 +1,25 @@
+<component name="libraryTable">
+  <library name="Dart SDK">
+    <CLASSES>
+      <root url="file://{{dartSdk}}/lib/async" />
+      <root url="file://{{dartSdk}}/lib/collection" />
+      <root url="file://{{dartSdk}}/lib/convert" />
+      <root url="file://{{dartSdk}}/lib/core" />
+      <root url="file://{{dartSdk}}/lib/developer" />
+      <root url="file://{{dartSdk}}/lib/html" />
+      <root url="file://{{dartSdk}}/lib/indexed_db" />
+      <root url="file://{{dartSdk}}/lib/io" />
+      <root url="file://{{dartSdk}}/lib/isolate" />
+      <root url="file://{{dartSdk}}/lib/js" />
+      <root url="file://{{dartSdk}}/lib/js_util" />
+      <root url="file://{{dartSdk}}/lib/math" />
+      <root url="file://{{dartSdk}}/lib/mirrors" />
+      <root url="file://{{dartSdk}}/lib/svg" />
+      <root url="file://{{dartSdk}}/lib/typed_data" />
+      <root url="file://{{dartSdk}}/lib/web_audio" />
+      <root url="file://{{dartSdk}}/lib/web_gl" />
+      <root url="file://{{dartSdk}}/lib/web_sql" />
+    </CLASSES>
```

Comment:
This file is scary. For example, why is web_sql in this list? Shouldn't dart:ui be there instead? When would the user edit this file?